### DISCUSSION TITLE
Vertical Section for Sliced Objects?

### DISCUSSION THREAD

**Discussion by Aurasphere-au** [OrcaSlicer/OrcaSlicer#7060] [General]
Im sure this has been asked before. When doing large models especially when skinned, this would be immensely helpful.

I searched and could not find it but surely its so common its been requested or is in the prog somewhere?

**Comment by otede** [comment]
I think you can only achieve it by temporarily using a Boolean/Modifier mesh to get rid of a large part of a model. BTW, the Discussions section seems to contain a lot of threads with 0 answers. No wonder people flock to Reddit.
